sql.txt

```diff
@@ -1,14 +1,35 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
+SELECT CustomerName, ContactName, Country FROM Customers ORDER BY country LIMIT 10 OFFSET 10;
+
 
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
+SELECT * FROM Customers WHERE ContactName LIKE 'a%'
+
 
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
+SELECT * FROM OrderDetails WHERE ProductID = 51 AND Quantity > 10;
+
 
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
 
-(5) Update the two top rows of the Products Table to increase the price by 1.50. (Get SQL to do the addition for you.)  Paste your SQL statement below.
+INSERT INTO Products (ProductID, ProductName, SupplierID, CategoryID, Unit, Price)
+VALUES (78, "JamieB", 42, 1, "10 bottles", 25);
+
+INSERT INTO Products (ProductID, ProductName, SupplierID, CategoryID, Unit,	Price)
+VALUES (79, "AL", 44, 2, "10 boxes", 25);  
+
+INSERT INTO Products (ProductID, ProductName, SupplierID, CategoryID, Unit,	Price)
+VALUES (81, "Jr", 2, 18, "30 lollipops", 25);
+
+(5)  ProductsUpdate the two top rows of the Table to increase the price by 1.50. (Get SQL to do the addition for you.)  Paste your SQL statement below.
+
+UPDATE Products SET Price = Price +1.50 WHERE ProductID IN (1, 2);
 
 (6) Delete all rows of the Products Table where the price is less than 7.00. Paste your SQL statement below.
+
+DELETE FROM Products WHERE Price < 7.00;
+
+TESTING DIFF HERE
```

Comment:
Pro tip: You can reopen your other pull request (that's closed), make changes and push them. That way you still have the context from the previous pull request and the diff is easier to read :)